src/witan/models/run_models.clj

```diff
@@ -102,10 +103,28 @@
       (data-csv/write-csv out-file
                           (convert-row-maps-to-vector-of-vectors ordered-data)))))
 
+(def local-authorities (read-string (slurp "resources/default_datasets/local_authorities.edn")))
+
+(defn get-district [gss-code]
+  (get-in local-authorities [:gss-code-to-district (keyword gss-code)]))
+
+(defn add-district-to-dataset-per-row [data] (wds/add-derived-column data :district [:gss-code] (fn [gss-code] (get-district gss-code))))
```

Comment:
Good point @elisehuard! That'd help with readability. But I think `cljfmt` will fix it.